src/FirePokemon.java

```diff
@@ -5,21 +5,21 @@
 
 
 public class FirePokemon extends Pokemon {
-    private List<String> attacks = Arrays.asList("inferno", "pyroBall", "fireLash", "flameThrower
+    private List<String> attacks = Arrays.asList("inferno", "pyroBall", "fireLash", "flameThrower");
```

Comment:
Je hebt twee lijsten met attacks. In de constructor maak je namelijk een nieuwe lijst. Waarom gebruik je deze daar niet? (denk aan DRY) 
De twee lijsten zijn ook niet hetzelfde. De ene heeft spaties en de ander niet. Het is voor de werking van de applicatie belangrijk dat je de aanval namen exact schrijft zoals ze ook in de PokemonGymImpl worden gebruikt.